lib/heroku/command/pg_backups.rb

```diff
@@ -513,12 +513,15 @@ def unschedule_backups
     db = shift_argument
     validate_arguments!
 
+    if db.nil?
+      abort("Must specify database to unschedule backups for")
+    end
```

Comment:
If there is only one schedule, should we accept no `db` here?

For this message, I'd add a hint: "Must specify a database. Run `heroku help pg:backups` for usage information."